S2VX.Game/EndGame/UserInterface/ScoreGrade.cs

```diff
@@ -0,0 +1,59 @@
+﻿using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics;
+
+namespace S2VX.Game.EndGame.UserInterface {
+    public class ScoreGrade : CompositeDrawable {
+        public string Grade { get; private set; } = "F";
+        public bool IsSquared { get; private set; }
+
+        public ScoreGrade(double accuracy, bool isFullCombo) {
+            DetermineGrade(accuracy, isFullCombo);
+
+            Origin = Anchor.Centre;
+            Position = new(250);
+            Size = new(500);
+
+            AddInternal(new SpriteText {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Text = Grade,
+                Font = new FontUsage(size: 200, weight: "bold"),
```

Comment:
Should this be a new TextSize constant (also line 33)?

Reply:
I decided not to do this since these constants are going to be unique for the score grade.